stdpopsim/slim_engine.py

```diff
@@ -421,8 +423,8 @@
     if (length(admixture_pulses) > 0 ) {
         for (i in 0:(ncol(admixture_pulses)-1)) {
             g = G_start + gdiff(T_start, admixture_pulses[0,i]);
-            dest = admixture_pulses[1,i];
-            src = admixture_pulses[2,i];
+            dest = asInteger(admixture_pulses[1,i]);
+            src = asInteger(admixture_pulses[2,i]);
```

Comment:
The new `asInteger` is because SLiM has changed how it prints out floats: it used to be that printing out `4.0` would produce `4`; but now it's `4.0`, which broke this code.